packs/data/feats.db/psychopomp-familiar.json

```
@@ -11,7 +11,7 @@
             "value": null
         },
         "description": {
-            "value": "<p>Your familiar becomes a masked psychopomp-an extraplanar guardian of the cycle of souls. This familiar continues to use all the same rules as other familiars, but one of its abilities must always be speech and it has the monitor and psychopomp traits. As normal when a familiar must always have a certain ability, the speech ability counts against your familiar's abilities each day.</p>\n<p>You can select three familiar or master abilities each day, instead of two, but one must be one of the following psychopomp familiar abilities.</p>\n<ul>\n<li><strong>Soul Sight</strong> Your familiar gains lifesense with a range of 30 feet.</li>\n<li><strong>Spirit Touch</strong> Your familiar can touch incorporeal creatures. If you have the spell delivery master ability from your familiar, any spell the familiar delivers with it gains the benefits of the <em>@Compendium[pf2e.equipment-srd.Ghost Touch]{Ghost Touch}</em> property rune.</li>\n</ul>"
+            "value": "<p>Your familiar becomes a masked psychopomp-an extraplanar guardian of the cycle of souls. This familiar continues to use all the same rules as other familiars, but one of its abilities must always be speech and it has the monitor and psychopomp traits. As normal when a familiar must always have a certain ability, the speech ability counts against your familiar's abilities each day.</p>\n<p>You can select three familiar or master abilities each day, instead of two, but one must be one of the following psychopomp familiar abilities.</p>\n<ul>\n<li>@Compendium[pf2e.familiar-abilities.Soul Sight]{Soul Sight}</li>\n<li>@Compendium[pf2e.familiar-abilities.Spirit Touch]{Spirit Touch}</li>\n</ul>"
```

```suggestion
            "value": "<p>Your familiar becomes a masked psychopomp-an extraplanar guardian of the cycle of souls. This familiar continues to use all the same rules as other familiars, but one of its abilities must always be speech and it has the monitor and psychopomp traits. As normal when a familiar must always have a certain ability, the speech ability counts against your familiar's abilities each day.</p>\n<p>You can select three familiar or master abilities each day, instead of two, but one must be one of the following psychopomp familiar abilities.</p>\n<ul>\n<li><strong>@Compendium[pf2e.familiar-abilities.Soul Sight]{Soul Sight}</strong> Your familiar gains lifesense with a range of 30 feet.</li>\n<li><strong>@Compendium[pf2e.familiar-abilities.Spirit Touch]{Spirit Touch}</strong> Your familiar can touch incorporeal creatures. If you have the spell delivery master ability from your familiar, any spell the familiar delivers with it gains the benefits of the <em>@Compendium[pf2e.equipment-srd.Ghost Touch]{Ghost Touch}</em> property rune.</li>\n</ul>"
```

This one is optional.